src/include/parser/explain_statement.h

```diff
@@ -0,0 +1,38 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// explain_statement.h
+//
+// Identification: src/include/parser/explain_statement.h
+//
+// Copyright (c) 2015-2018, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "parser/sql_statement.h"
+
+namespace peloton {
+
+class SqlNodeVisitor;
+
+namespace parser {
+
+/**
+ * @class ExplainStatement
+ * @brief Represents "EXPLAIN <query>"
+ */
+class ExplainStatement : public SQLStatement {
+ public:
+  ExplainStatement() : SQLStatement(StatementType::EXPLAIN) {}
+  virtual ~ExplainStatement() {}
+
+  virtual void Accept(UNUSED_ATTRIBUTE SqlNodeVisitor *v) override {}
```

Comment:
1. Any reason why this isn't implemented?
2. You can remove the `virtual` keyword. It's redundant since you specify `override`.

Reply:
1) I could implement it like other *Statement classes by inserting a `v->Visit(this)` but all the statement classes currently doesn't implement `Visit` function. So, implementing it here would be a no-op. Anyhow, I am implementing it like other statements just for consistency sake.

Reply:
2) This use of `virtual` keyword is inspired by other such *Statement classes in the code. Sure, I can remove it but I left it there for consistency sake, again. I am removing it in my latest patchset.